jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -73,6 +80,55 @@ local excludedRules = [
       { alert: 'PrometheusErrorSendingAlertsToAnyAlertmanager' },
     ],
   },
+  {
+    name: 'kube-prometheus-node-recording.rules',
+    rules: [
+      { alert: 'instance:node_cpu:rate:sum' },
+      { alert: 'instance:node_network_receive_bytes:rate:sum' },
+      { alert: 'instance:node_cpu:ratio' },
+      { alert: 'instance:node_network_transmit_bytes:rate:sum' },
+    ],
+  },
+  {
+    name: 'node.rules',
+    rules: [
+      { record: 'node:node_num_cpu:sum' },
+      { record: 'node_namespace_pod:kube_pod_info:' },
+    ],
+  },
+  {
+    name: 'openshift-build.rules',
+    rules: [
+      { record: 'build_error_rate' },
+    ],
+  },
+  {
+    name: 'openshift-kubernetes.rules',
+    rules: [
+      { record: 'kube_running_pod_ready' },
+      { record: 'namespace:container_cpu_usage:sum' },
+      { record: 'namespace:container_memory_usage_bytes:sum' },
+      { record: 'namespace:container_spec_cpu_shares:sum' },
+      { record: 'pod:container_cpu_usage:sum' },
+      { record: 'pod:container_fs_usage_bytes:sum' },
+      { record: 'pod:container_memory_usage_bytes:sum' },
+      { record: 'pod:container_spec_cpu_shares:sum' },
+    ],
+  },
+  {
+    name: 'openshift-ingress.rules',
+    rules: [
+      { alert: 'code:cluster:ingress_http_request_count:rate5m:sum' },
```

Comment:
it isn't removed from assets?

Reply:
I mistook a record for alert in the rule definition😅